### DISCUSSION TITLE
Please remove malicious malware trends on the platform! (e.g. "4K-Video-Downloader-Crack")

### DISCUSSION THREAD

**Discussion by Animan8000** [community/community#60322] [Other Feature Feedback, Questions, & Ideas]
Since over 1-2 months GitHub has been recommending malware about fake 4K-Video-Downloader Cracked repositories constantly to me. I don't even use that software since years, also because yt-dlp is endlessly more powerful in comparison. These trends including repositories obviously don't just involve potential piracy, but to myself they look like an attempt to infect innocent users on the platform. The huge problem is that this nonsense is constantly getting recommended to me all the time despite me not using the related original software since forever, see screenshots (both of them are from today)
![Screenshot_20230707_140737](https://github.com/community/community/assets/30600651/94efc030-0b0c-4345-b35c-f59fa552deb7)
![Screenshot_20230707_141008](https://github.com/community/community/assets/30600651/2203cb4c-5e3a-437e-8350-fdf6f40984e1)

So in other words: GitHub recommends to me constantly malicious trends of the same software, except that software is infected with malware extremely likely and fake/infected accounts star these repositories to push these recommendations to other people, exploiting the whole algorithm to infect more innocent people on the platform. Please do something about that!
